aiohttp/helpers.py

```diff
@@ -115,6 +116,25 @@ def strip_auth_from_url(url):
 
 
 def proxies_from_env():
+    netrc_obj = None
+    netrc_path = os.environ.get('netrc')
```

Comment:
Environment names are case sensitive, according to https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html the name should be upper-cased: `NETRC`.